src/app/quiz/questions.js

```diff
@@ -0,0 +1,77 @@
+export const questions = [
+    {
+      question: 'What is JavaScript?',
+      options: [
+        'A programming language',
+        'A markup language',
+        'A database',
+        'An operating system'
+      ],
+      correct: 0
+    },
+    {
+      question: 'Which keyword is used to declare variables in JavaScript?',
+      options: ['var', 'let', 'const', 'All of the above'],
+      correct: 3
+    },
+    {
+      question: 'What is the result of 2 + "2" in JavaScript?',
+      options: ['4', '22', 'NaN', 'Error'],
+      correct: 1
+    },
+    {
+      question: 'What is the purpose of the "use strict" directive in JavaScript?',
+      options: [
+        'To enforce stricter parsing and error handling',
```

Comment:
I don't think there's a correct answer that would fit on one screen.

Reply:
ok, we can change that